.buildkite/release_pipeline.yaml

```diff
@@ -170,6 +227,14 @@ steps:
     command: .buildkite/scripts/bintray_upload.sh
     agents:
       queue: habitat-release
+  
+  - label: ":linux: Upload (but don't publish!) artifacts to Bintray"
```

Comment:
Just to clarify: uploading to Bintray for kernel2 is just easier if we run the script again with a different `BUILD_PKG_TARGET`, right? Is there any benefit at all to modifying the overall script to try and upload _all_ Linux packages in a single operation?

Reply:
Reading the source for bintray_upload, it uses `hab pkg install` to get the artifact on disk and then call bintray-publish.  Since that will only install for the ActiveTarget, I elected to split the steps out, similar to the Mac step. 

I think we'll want to update that to use curl instead, which would allow us to collapse all targets to a single "bintray upload" step.  I felt that that was a larger change that should be separately from this change though.

I'm not sure I understand the BUILD_PKG_TARGET question.  If I remember correctly, that only comes in to play when you're bootstrapping the hab ecosystem and are executing plan-build out of the source tree, rather than from a package.

Reply:
I think your approach is a good one; I was just wanting to confirm my suspicion.

I also think that we're probably at the point where we could likely replace the bintray package with a shell script in the pipeline and call it a day. But I agree, that's a bigger change for a later time.

I'm 👍 on what you're doing here.